File: MyHordesOptimizerApi/MyHordesOptimizerApi/Dtos/MyHordesOptimizer/CitizenState/CitizenStateStepDto.cs
namespace MyHordesOptimizerApi.Dtos.MyHordesOptimizer.CitizenState
{
    public class CitizenStateStepDto
    {
        /// <summary>"item", "move", "equip_shoes", "mount_bike", "dismount_bike", "pickup_defence_cp_item" ou "become_ghoul".</summary>
        public string Type { get; set; } = null!;
        public int? ItemId { get; set; }
        public bool? IsNearZone { get; set; }
    }
}

File: MyHordesOptimizerWebsite/src/app/_abstract_model/dto/citizen-state-step.dto.ts
export interface CitizenStateStepDTO {
    type: 'item' | 'move' | 'equip_shoes' | 'mount_bike' | 'dismount_bike' | 'pickup_defence_cp_item' | 'become_ghoul';
    itemId?: number;
    isNearZone?: boolean;
}
